tests/PHPStan/Analyser/nsrt/instanceof.php

```diff
@@ -80,9 +80,9 @@ public function testExprInstanceof($subject, string $classString, $union, $inter
 			assertType('true', $subject instanceof Foo);
 			assertType('bool', $subject instanceof $classString);
 		} else {
-			assertType('mixed~InstanceOfNamespace\Foo', $subject);
-			assertType('false', $subject instanceof Foo);
-			assertType('false', $subject instanceof $classString);
+			assertType('mixed', $subject);
```

Comment:
if I pass `class-string<SuperFoo>` for `$classString` the subject will only be `mixed~SuperFoo` but not `mixed~Foo`.